packages/cmf/src/sagas/http.js

```diff
@@ -111,6 +117,74 @@ export function httpFetch(url, config, method, payload) {
 		.then(handleHttpResponse)
 		.catch(handleError);
 }
+export function cloneError(initialError, resultError) {
+	const errorToEnrich = Object.assign({}, resultError);
+	return initialError.stack.response
+		.clone()
+		.text()
+		.then(response => {
+			try {
+				errorToEnrich.stack.response = response;
+				errorToEnrich.stack.messageObject = JSON.parse(response);
+				return Promise.resolve(errorToEnrich);
+			} catch (exception) {
+				errorToEnrich.stack.messageObject = 'Parsing response error';
+				return Promise.reject(errorToEnrich);
+			}
+		});
+}
+export function* errorProcessing(error, httpAction) {
+	const errorObject = {
+		name: error.name,
+		message: error.description || error.message,
+		number: error.number,
+		stack: error.stack,
+	};
+	const clone = get(error, 'stack.response.clone');
+	if (!clone) {
+		yield put(onJSError(errorObject, httpAction));
```

Comment:
I would like to add a console.error here. What do you think about this one ?
Most of the time it's an error in a reducer we just catch.
So we are sure the dev see this in the console which is not the case at the moment

Reply:
:+1: I agree on providing an explicit message to developer